.github/kokoro/install.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+SCRIPT_SRC="$(realpath ${BASH_SOURCE[0]})"
+SCRIPT_DIR="$(dirname "${SCRIPT_SRC}")"
+INSTALL_DIR="$(pwd)/install"
+GIT_DESCRIBE=$(git describe)
+
+export CMAKE_FLAGS="-GNinja -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR}"
+export BUILD_TOOL=ninja
+source ${SCRIPT_DIR}/common.sh
+
+echo
+echo "========================================"
+echo "Running install tests (make install)"
+echo "----------------------------------------"
+(
+	pushd build
+	export VPR_NUM_WORKERS=${CORES}
+	ninja -j${MAX_CORES} install
+	popd
+)
+echo "----------------------------------------"
+
+echo
+echo "========================================"
+echo "Compressing and uploading install dir"
+echo "----------------------------------------"
+(
+	tar -c --use-compress-program="pigz" -f symbiflow-arch-defs-install-${GIT_DESCRIBE}.tar.gz ${INSTALL_DIR}
```

Comment:
Why not bz2 or lzma?

Reply:
I have found pigz to be the fastest one, with a good compression ratio. 

The local install directory I have is ~12 Gb, while the compressed archive is ~2.5 Gb

Reply:
Can you post your results table?  I'm willing to wait an hour to gain a large compression ratio drop.  Speed is not important here.

Reply:
Sure, I am capturing exact data now.

Reply:
@litghost This is what I have for now:

```
total uncompressed size: 12.2G

pigz:
    real    0m31.171s
    user    9m9.889s
    sys     0m11.734s

    compressed size: 2.5G
    compression ratio: 4.88
    space saving: 79.51%

bz2:
    real    15m16.055s
    user    15m12.854s
    sys     0m21.321s

    compressed size: 1.9G
    compression ratio: 6.4211
    space saving: 84.43%

lzma:
    real    46m39.288s
    user    46m43.604s
    sys     0m52.514s

    compressed size: 852M
    compression ratio: 14.6714
    space saving: 93.18%

```

I am running also `xz` compression which has the multithread implementation as well. Will update as soon as it finishes.

Reply:
LZMA is definitely the way to go here.

Reply:
Actually found xz to be much faster with even better compression ratio than LZMA:

```
xz:
    real    5m47.481s
    user    91m35.688s
    sys     0m20.094s

    compressed size: 850M
    compression ratio: 14.7059
    space saving: 93.2%
```

Reply:
Works for me!

Reply:
> Question: how does the install kokoro cfg get triggered? I cannot see it running

Working on it

Reply:
> > Question: how does the install kokoro cfg get triggered? I cannot see it running
> 
> Working on it

Should be working in 20 minutes, hopefully.